app/helpers/file_helper.rb

```diff
@@ -187,11 +187,11 @@ def compress_image(path)
       exec = "#{Rails.root.join('lib', 'shell', 'timeout.sh')} -t 30 nice -n 10 convert \"#{path}\" -resize 1024x1024 \"#{tmp_file}\" >>/dev/null 2>>/dev/null"
       # puts exec
 
-      # try with convert
-      did_compress = false
-      Terminator.terminate 40 do
-        did_compress = system exec
-      end
+    # try with ghostscript
+    did_compress = false
+    try_within 120, "compressing image" do
```

Comment:
Could you change these to the newer shorter times... 2 minutes ended up being too much here. 40 sec now... though I think it will be less later.